README.rst

```diff
@@ -4,6 +4,4 @@ construction of our website and documentation.
 This repository is usually pulled into the appropriate places as a git
 submodule.
```

Comment:
This is probably not useful. We could say something like "Sphinx extensions to support the Dylan programming language." instead.  (optional)